experimentation/dashboards/experiment_enrollments.dashboard.lookml

```diff
@@ -557,7 +557,7 @@
     dynamic_fields:
     - category: table_calculation
       expression: "${events.event_count} * 100"
-      label: approx count
+      label: Adjusted Event Count
```

Comment:
IMO "Approximate Event Count" would give a better impression of the nature of this value.